lua/neotest-go/utils.lua

```diff
@@ -157,15 +157,22 @@ function utils.get_errors_from_test(test, file_name)
 end
 
 ---@param tree neotest.Tree
----@param name string
 ---@return string
-function utils.get_prefix(tree, name)
-  local parent_tree = tree:parent()
-  if not parent_tree or parent_tree:data().type == "file" then
```

Comment:
So we don't need that check anymore, do we?

Reply:
Which check do you mean?